ansible/roles/postgresql/tasks/main.yml

```diff
@@ -60,6 +60,14 @@
 - name: assert hot standby config correct
   assert: { that: "hot_standby_server is not defined or hot_standby_server in groups.pg_standby"}
 
+- name: Create postgresql_log_directory
+  file:
+    dest: "{{ postgresql_log_directory }}"
+    state: directory
+    owner: postgres
+    group: postgres
+  when: postgresql_log_directory|default(false)
```

Comment:
why is this necessary? this seems unrelated to pg_backrest

Reply:
following advise on https://github.com/pgbackrest/pgbackrest/issues/413
just to save disk space